manifests/s/SuperCuber/Dotter/0.13.1/SuperCuber.Dotter.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SuperCuber.Dotter
+PackageVersion: 0.13.1
+InstallerType: portable
+Commands:
+- dotter

```

Comment:
```suggestion
- dotter
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Reply:
Oh wow! I had no idea winget could specify dependencies like that. That would be perfect!